### DISCUSSION TITLE
Link vertex

### DISCUSSION THREAD

**Discussion by snehabihani** [Blazor-Diagrams/Blazor.Diagrams#210] [Q&A]
Hi Can we add our own custom razor component instead of circle for link vertices?

**Comment by zHaytam** [comment]
Hello, current no sorry.
What you can do is copy the code for `LinkWidget` and change line 62 ([here](https://github.com/Blazor-Diagrams/Blazor.Diagrams/blob/master/src/Blazor.Diagrams/Components/LinkWidget.razor)) in order to use whatever component you want!

**Comment by snehabihani** [comment]
I tried.. But it is not getting displayed.. I am not sure why?

**Comment by zHaytam** [comment]
You'll need to show me what you've done so that I can help.

**Reply by snehabihani** [reply]
[LinkWrapperWidget.zip](https://github.com/Blazor-Diagrams/Blazor.Diagrams/files/9120190/LinkWrapperWidget.zip)

**Reply by snehabihani** [reply]
Please check line number 61 in widget. right now i have added hardcode px for div in nodewidgetcomponent

**Reply by zHaytam** [reply]
If you want HTML as your vertices, you need to wrap it in a `foreignObject`, otherwise HTML is ignored inside SVG.
